src/products/components/ProductMedia/ProductMedia.tsx

```diff
@@ -209,42 +219,87 @@ const ProductImages: React.FC<ProductImagesProps> = props => {
     <Card className={classes.card}>
       <CardTitle
         title={intl.formatMessage({
-          defaultMessage: "Images",
+          defaultMessage: "Media",
           description: "section header"
         })}
         toolbar={
           <>
             <Button
-              onClick={() => upload.current.click()}
-              disabled={loading}
+              onClick={() => setPopperOpenStatus(true)}
               variant="text"
               color="primary"
               data-test="button-upload-image"
+              ref={anchor}
             >
-              {intl.formatMessage(commonMessages.uploadImage)}
+              {intl.formatMessage({
+                defaultMessage: "Upload",
+                description: "modal button upload"
+              })}
             </Button>
+
+            <Popper
```

Comment:
This component is long already. I think the Popper along with contents could be extracted to a separate component